site/layouts/partials/analytics.html

```diff
@@ -4,3 +4,4 @@
 {{- end }}
 {{- $secureTarteJS := $tarteJS | resources.Fingerprint "sha384" }}
 <script src="{{ $tarteJS.Permalink | absURL }}" {{ printf "integrity=%q" $secureTarteJS.Data.Integrity | safeHTMLAttr }} crossorigin="anonymous"></script>
+<!-- Boosted mod: no Fathom Analytics (yet ?) -->
```

Comment:
Maybe we could create an issue to study if it would be interesting to also use Fathom Analytics

Reply:
Done !